### PR TITLE
Renamed "SIZE" text controller

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
Both, the label and the comment for the text controller `SIZE` are incorrect.
I don't think it changes the size specifically.
After fiddling with it, I feel like it changes the font for the current text string.
There isn't one single valid value to use with it either *(`SMALL`)*, there's 5 for the English font, and 3 for the Japanese font, I'd say.

Maybe some constants should be added for these values.

## **Discord contact info**
Lunos#4026

### CODE REVIEW

**Comment by GriffinRichards** [comment]
Could you rename the corresponding C define as well? The one here:
https://github.com/pret/pokeemerald/blob/6dc3c9864c80557a81c080e0a5893d9d8765c635/gflib/text.h#L214

**Comment by LOuroboros** [comment]
> 
> 
> Could you rename the corresponding C define as well? The one here:
> 
> https://github.com/pret/pokeemerald/blob/6dc3c9864c80557a81c080e0a5893d9d8765c635/gflib/text.h#L214

Done :eyes:

File: charmap.txt [hunk 1]
```diff
@@ -414,7 +414,7 @@ HIGHLIGHT = FC 02 @ same as fc 01
 SHADOW = FC 03 @ same as fc 01
 COLOR_HIGHLIGHT_SHADOW = FC 04 @ takes 3 bytes
 PALETTE = FC 05 @ used in credits
-SIZE = FC 06 @ note that anything other than "SMALL" is invalid
+FONT = FC 06 @ valid values are 0, 1, 2, 7 and 8
```
Comment (GriffinRichards) [inline charmap.txt:417]: Oops, sorry to not lump the comments together, but looks like 6 is a valid font ID as well (for Braille)

Comment (LOuroboros) [reply charmap.txt:417]: Should I modify the comment like "0, 1, 2, 6, 7 and 8" or "0, 1, 2, 6 (braille), 7 and 8"? :eyes:

Comment (GriffinRichards) [reply charmap.txt:417]: Sure. If we end up adding font id constants later we'll probably replace those numbers with the constant names, or a note on where to find them.

Comment (LOuroboros) [reply charmap.txt]: > Whoops by sure I meant the latter lol

Done :eyes:

**Pull request merged**